notebooks/timeseries.ipynb

```diff
@@ -0,0 +1,568 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# OpenSCM Timeseries\n",
+    "\n",
+    "In OpenSCM, the class for handling timeseries is `openscm.timeseries.Timeseries`. \n",
+    "\n",
+    "A timeseries in OpenSCM is always initialised from arrays i.e. discrete data which are associated with specific points in time. In this notebook we call the initialisation values the 'source data' and the times with which they are associated the 'source times'.\n",
+    "\n",
+    "To estimate what the value of the variable under consideration is at points in time other than the source points (call these points the 'target times'), i.e. to interpolate or extrapolate, we always take the following steps:\n",
+    "\n",
+    "1. start with the data which was originally used to set the timeseries\n",
+    "1. convert the discrete data to continuous data using an interpolation/extrapolation technique\n",
+    "    - e.g. linear interpolation, cubic spline, moving average\n",
+    "1. use the continuous data to determine the values at the target times aka the 'target values'\n",
```

Comment:
```suggestion
    "3. use the continuous data to determine the values at the target times aka the 'target values'\n",
```

Reply:
This should render as a 3 automatically?